js/main.js

```diff
@@ -29,5 +32,97 @@ function submitForm(event) {
   data.entries.unshift(inputValues);
   data.nextEntryId++;
   newEntryForm.reset();
+
+  var newEntry = renderEntry(data.entries[0]);
+  ulElement.prepend(newEntry);
+
+  entriesView.className = 'view';
+  entryFormView.className = 'view hidden';
+  data.view = 'entries';
   img.setAttribute('src', 'images/placeholder-image-square.jpg');
 }
+
+var body = document.querySelector('body');
+var allView = document.querySelectorAll('.view');
+
+function renderEntry(entry) {
+  var entryItem = document.createElement('li');
+  var divRow = document.createElement('div');
+  divRow.setAttribute('class', 'row');
+  entryItem.appendChild(divRow);
+  var columnHalfImage = document.createElement('div');
+  columnHalfImage.setAttribute('class', 'column-half column-entry entry-image');
+  divRow.appendChild(columnHalfImage);
+  var imgRender = document.createElement('img');
+  imgRender.setAttribute('src', entry.url);
+  columnHalfImage.appendChild(imgRender);
+  var columnHalfText = document.createElement('div');
+  columnHalfText.setAttribute('class', 'column-half column-entry entry-text');
+  divRow.appendChild(columnHalfText);
+  var h3 = document.createElement('h3');
+  h3.setAttribute('class', 'font-proza title-styling');
+  columnHalfText.appendChild(h3);
+  h3.textContent = entry.title;
+  var p = document.createElement('p');
+  columnHalfText.appendChild(p);
+  p.textContent = entry.notes;
+  return entryItem;
+}
+
+document.addEventListener('DOMContentLoaded', loadedPage);
+
+function loadedPage(event) {
+  var ulElement = document.querySelector('ul');
+
+  for (var i = 0; i < data.entries.length; i++) {
+    var domTree = renderEntry(data.entries[i]);
+    ulElement.appendChild(domTree);
+  }
+}
+
+body.addEventListener('click', switchView);
+
+function switchView(event) {
```

Comment:
You shouldn't make this function reliant on an event object. You want this to have a parameter named something like `view` which is a string of the name of the view to go to. That way, you can use this function to switch between views in any context, not just an event handler context 🤓